java/vector/src/main/codegen/data/ValueVectorTypes.tdd

```diff
@@ -56,8 +56,10 @@
         { class: "Int", valueHolder: "IntHolder"},
         { class: "UInt4", valueHolder: "UInt4Holder" },
         { class: "Float4", javaType: "float" , boxedType: "Float", fields: [{name: "value", type: "float"}]},
-        { class: "IntervalYear", javaType: "int", friendlyType: "Period" }
-        { class: "Time", javaType: "int", friendlyType: "DateTime" }
+        { class: "DateDay", javaType: "int", boxedType: "Integer", friendlyType: "DateTime" },
```

Comment:
You can add constructor just for the time/date/timestamp type that takes a timeUnit and store in the vector object